app/qml/inputs/MMTextBubblesEditor.qml

```diff
@@ -0,0 +1,168 @@
+
+  Component.onCompleted: root.hideFeaturesIfNeeded()
+  onWidthChanged: root.hideFeaturesIfNeeded()
+
+  content: Rectangle {
+    width: root.width - 2 * root.spacing
+    height: root.contentItemHeight
+    color: __style.whiteColor
+
+    Flow{
+      id: flow
+
+      anchors.fill: parent
+      anchors.margins: 10 * __dp
+      spacing: 8 * __dp
+      clip: true
+
+      Rectangle {
+        width: 40 * __dp
+        height: root.itemHeight
+        radius: 8 * __dp
+        color: __style.lightGreenColor
+
+        MMIcon {
+          anchors.centerIn: parent
+          source: __style.plusIcon
+          useCustomSize: true
+          width: 16 * __dp
+          height: width
+        }
+
+        MouseArea {
+          anchors.fill: parent
+          onClicked: root.createLinkedFeature( root.parent.featurePair, root.parent.associatedRelation )
+        }
+      }
+
+      Repeater {
+        id: repeater
+
+        property var invisibleIds: []
+
+        model: root.featuresModel
+
+        delegate: Rectangle {
+          width: text.contentWidth + 24 * __dp
+          height: root.itemHeight
+          radius: 8 * __dp
+          color: __style.mediumGreenColor
+          visible: {
+            let isVisible = (y < 2 * root.itemHeight || (y < 3 * root.itemHeight && x + width < flow.width - footer.width - flow.spacing))
+            if(!isVisible && !repeater.invisibleIds.includes(model.FeatureId))
+              repeater.invisibleIds.push(model.FeatureId)
+            footer.visible = repeater.invisibleIds.length > 0
+            moreText.text = ("+" + repeater.invisibleIds.length + qsTr(" more"))
+            return isVisible
+          }
+
+          Text {
+            id: text
+
+            anchors.centerIn: parent
+            font: __style.t4
+            text: model.FeatureTitle
+            color: __style.forestColor
+            horizontalAlignment: Text.AlignHCenter
+            verticalAlignment: Text.AlignVCenter
+          }
+
+          MouseArea {
+            anchors.fill: parent
+            onClicked: root.openLinkedFeature( model.FeaturePair )
+          }
+        }
+      }
+
+      Rectangle {
+        id: footer
+
+        width: 100 * __dp
+        height: root.itemHeight
+        radius: 8 * __dp
+        color: __style.lightGreenColor
+
+        Text {
+          id: moreText
+
+          anchors.centerIn: parent
+          font: __style.t4
+          color: __style.forestColor
+          horizontalAlignment: Text.AlignHCenter
+          verticalAlignment: Text.AlignVCenter
+        }
+
+        MouseArea {
+          anchors.fill: parent
+          onClicked: {
+            if ( !root.enabled )
+              return
+            listLoader.active = true
+            listLoader.focus = true
+          }
+        }
+      }
+    }
+  }
+
+  function hideFeaturesIfNeeded() {
+    repeater.invisibleIds = []
+    repeater.model = undefined
```

Comment:
It was `null` previously, not sure if it makes a difference

```suggestion
    repeater.model = null
```

Reply:
It looks, behavior is the same